.github/workflows/reviewdog-workflow.yml

```diff
@@ -160,21 +194,17 @@ jobs:
           # - shellcheck_flags
 
   wemake-python-styleguide:
+    needs: pre_job_python_determinator
+    if: ${{ needs.pre_job_python_determinator.outputs.should_not_skip == 'true' }}
     name: wemake-python-styleguide
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
         with:
-          fetch-depth: 0
           ref: ${{github.event.pull_request.head.ref}}
           repository: ${{github.event.pull_request.head.repo.full_name}}
-      - name: Get changed files
-        id: py-changes
-        # Set outputs.py to be a list of modified python files
-        run: |
-          echo "::set-output name=py::$(git diff --name-only --diff-filter=ACMRT ${{ github.event.pull_request.base.sha }} ${{ github.event.pull_request.head.sha }} | grep .py$ | xargs)"
```

Comment:
There were the following issues with this part, such that I replaced it.
- Errors occurred on this line.
- It is not reusable for other actions.
- It is not consistent with the common approach of dorny-paths-filter.

Reply:
If that line is prone to errors and you have a refactoring, you might also want to look at: https://github.com/magma/magma/blob/53c4848439f1a8255a2d3f63219900a8e1e1ac9b/.github/workflows/python-workflow.yml#L63

Reply:
Good catch. However, overall a more general refactoring and consolidation of linter and formatter application might be beneficial, which I would however see outside of the scope of this PR. :wink: